odoo/addons/base/models/ir_actions_report.py

```diff
@@ -410,6 +413,58 @@ def _prepare_html(self, html, report_model=False):
 
         return bodies, res_ids, header, footer, specific_paperformat_args
 
```

Comment:
We could consider that if wkhtmltopdf is installed, so is wkhtmltoimage as packages of major distros ship both in the same package. But it could make sense for people who try to have minimal installs, will add a check.